README.md

```diff
@@ -69,6 +69,18 @@ And trigger the process using:
 
     VERSION_TAG=myfeature sbt ecr:push
 
+## Cross account
+
+By default, When the login task is executed, authentication will target the **registry id** 
+and **repository domain** of the AWS account belonging to the role used.
+
+If you need cross account authentication, 
+you can override registry domain and target any registry id.
+
+Example usage:
+    repositoryDomain in Ecr := "your ecr repository domain"
+    registryIds in Ecr ++= Seq("your AWS account id")
```

Comment:
1. Please make sure, that the README file is formatted correctly (https://markdownlivepreview.com/):
- An empty line is needed after `Example usage:`. 
- Sentence in the previous paragraph can probably be broken in a more natural way (with the `80` chars limit in place) => detail.
2. I would use more "natural" examples, that imply a proper format: 
- `"your ecr repository domain"` => `"myecr.example.com"`